AuxTel/AuxTel-Troubleshooting/LATISS-Troubleshooting/ATcamera-recovery.rst

```diff
@@ -0,0 +1,160 @@
+.. note::
+    The instructions below assume:
+        #. The ability to login to the AuxTel CCS computers, 
+        #. Some familiarity with basic CCS commands/functionality. 
+
+    We need a separate document to provide this background information since it will need to be referred to from multiple places.
+
+
+.. _ATcamera-recovery/Error-Diagnosis:
+
+Error diagnosis
+===============
+
+.. This section should provide simple overview of known or suspected causes for the error.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Post screenshots of the error state or relevant tracebacks.
+
+.. Added error diagnosis
+- ATCamera goes to `FAULT` state.
+
+.. _ATcamera-recovery/Procedure-Steps:
+
+Procedure Steps
+===============
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+.. In general the steps are:
+#. Identify which **CCS subsystem triggered** the problem.
+#. Review the **raised alerts and/or log files**, and determine IF:
+    #. This was a transitory problem which can be documented (via JIRA ticket) and reset, 
+    #. or something which requires a camera expert to diagnose. 
+    #. Clear the raised alerts in both the CCS subsystem which triggered the problem and the Master Control Module (MCM) which tracks the overall camera state.
+#. Clear the fault in the ocs-bridge, and switch it back of OFFLINE_AVAILABLE mode.
+
+.. note::
+    In either case it is important that an **OBS ticket** be created so we can track how often specific problems occur, and whether software or hardware changes are needed to prevent future occurrences.
+
+Specific CCS commands for performing these operations are documented below.
+
+
+.. _ATcamera-recovery/Tracking-down-a-CSC-problem:
+
+Tracking down a CSC problem
+--------------------------------
+In general there are two approaches on tracking down a CCS problem, 
+either using the **ccs-shell** command line tool, or using the **ccs-console** graphical interface. 
+Currently we describe only the first approach.
+
+
+
+.. warning::  
+    Pending **TODO**: Simulate a fault and verify these commands are correct (perhaps on TTS) (plus highlight responses)
+
+.. this note was added to be able to copy the commands without cs. BUT UNCERTATIN WHETHER IF IT'S CORRECT OR NEEDED.
+
+.. admonition:: Important
```

Comment:
This is great! Now we need a procedure that tells us how to connect to the CCS :)